### DISCUSSION TITLE
Pregunta lab 04.01 task 3

### DISCUSSION THREAD

**Discussion by Andr3sPA** [rramosp/ai4eng.v1#283] [04 - GESTIÓN DE DATOS]
![image](https://github.com/rramosp/ai4eng.v1/assets/82219767/5847781f-a21b-425b-965e-4c00d1b06b47)
No sé cual es el problema con mi código, ¿entendí mal y así no es la formula?

**Comment by Felipe-RA** [comment]
Hola @Andresdar 

Mi sospecha inicial es que la aplicación de la función lambda no es correcta.

Ponme una copia de tu notebook para echarle un vistazo mas a fondo y poderte dar una respuesta con más seguridad.

Gracias.

**Reply by Andr3sPA** [reply]
ya lo solucioné, escribí r2 en vez de r3.

**Reply by Felipe-RA** [reply]
Vaya... es cierto!
